pkg/driver/device_manager.go

```diff
@@ -34,10 +34,12 @@ type DeviceManager interface {
 	MaxMetricSize() int
 	// MaxRequestsSize return the default maximum request logs size in bytes for this device manager
 	MaxRequestsSize() int
+	// MaxAppLogsSize return the default maximum app logs size in bytes for this device manager
+	MaxAppLogsSize() int
 	// Database safe-to-print (without credentials) path to database
 	Database() string
 	// Init initialize the datastore. If the given database URL is invalid for this type of manager, return false. Return error for actual failures
-	Init(string, int, int, int, int) (bool, error)
+	Init(string, int, int, int, int, int) (bool, error)
```

Comment:
This is fine for now, but we will have to start making this `Init()` saner (in a future PR), maybe passing it a `struct`